docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: '3'
+services:
+
+  fdk-model-publisher:
+    build: .
+    image: eu.gcr.io/digdir-fdk-infra/fdk-model-publisher:test
+    ports:
+      - "8080:8080"
+    environment:
+      - RABBIT_USERNAME=admin
+      - RABBIT_PASSWORD=admin
+      - RABBIT_HOST=rabbitmq
+      - FDK_DATASERVICE_HARVESTER=https://dataservices.staging.fellesdatakatalog.digdir.no
+      - FDK_MODEL_PUBLISHER_URI=https://fdk-model-publisher.fellesdatakatalog.no
+    depends_on:
+      - fdk-publishers-cache
+      - rabbitmq
+
+  fdk-publishers-cache:
```

Comment:
denne trenger ikke komplisert navn - redis holder massevis

Reply:
Vel ja, men dette er navnet den har i gcp. Om vi bruker redis som navn for lokal utvikling så må jeg legge navnet til som en miljøvariabel for å dynamisk kunne endre config. Tenkte det var enklere hardkode config med fdk-publishers-cache